src/metabase/models/database.clj

```diff
@@ -34,23 +34,67 @@
 
 (models/defmodel Database :metabase_database)
 
+
+(defn- schedule-tasks!
+  "(Re)schedule sync operation tasks for DATABASE. (Existing scheduled tasks will be deleted first.)"
+  [database]
+  (try
+    (require 'metabase.task.sync-databases)
+    ((resolve 'metabase.task.sync-databases/schedule-tasks-for-db!) database)
+    (catch Throwable e
+      (log/error "Error scheduling tasks for DB:" (.getMessage e) "\n")
```

Comment:
TBH I've never seen/used `log/errorf`. I'll read into it sounds like a good idea